commerce_coordinator/apps/lms/utils.py

```diff
@@ -0,0 +1,19 @@
+"""LMS Utility Functions"""
+
+from commerce_coordinator.apps.commercetools.clients import CommercetoolsAPIClient
+
+
+def get_line_item_from_entitlement(order_number: str, entitlement_id: str) -> str:
+    """
+    Check if line item with given entitlement is present.
+    """
```

Comment:
```suggestion
  """
    Retrieve the order ID and line item ID associated with the given entitlement ID.

    Args:
        order_number (str): The order number in Commercetools.
        entitlement_id (str): The entitlement ID to search for.

    Returns:
        tuple[str, str]: A tuple containing the order ID and the matching line item ID (if found, otherwise an empty string).
    """
```